app/javascript/packages/rails-i18n-webpack-plugin/rails-i18n-webpack-plugin.spec.js

```diff
@@ -96,23 +96,23 @@ describe('RailsI18nWebpackPlugin', () => {
             'actual1.en.js',
             'actual1.es.js',
             'actual1.fr.js',
-            'actual452.en.js',
-            'actual452.es.js',
-            'actual452.fr.js',
-            'actual946.js',
-            'actual946.en.js',
-            'actual946.es.js',
-            'actual946.fr.js',
+            'actual278.en.js',
+            'actual278.es.js',
+            'actual278.fr.js',
+            'actual910.js',
+            'actual910.en.js',
+            'actual910.es.js',
+            'actual910.fr.js',
           ]);
           expect(manifest.entrypoints['2'].assets.js).to.include.all.members([
             'actual2.js',
-            'actual452.en.js',
-            'actual452.es.js',
-            'actual452.fr.js',
-            'actual946.js',
-            'actual946.en.js',
-            'actual946.es.js',
-            'actual946.fr.js',
+            'actual278.en.js',
```

Comment:
is it a valuable test to hardcode the specific files? would it make sense to have like a count + regex approach?

Reply:
I'll check again in the morning if there's a strong reason to test them specifically. I think I wasn't expecting the suffix to change all that often since it's meant to be deterministic, but this isn't the first time I've had to update these as part of a version upgrade.

Reply:
I updated this in 6452166d598299892a5a95c5f4d179f4cbd852d2 to be hopefully more resilient to these sorts of deterministic ID changes.